src/integrationTest/java/uk/gov/hmcts/darts/task/service/UnstructuredToArmBatchProcessorIntTest.java

```diff
@@ -92,6 +84,7 @@ class UnstructuredToArmBatchProcessorIntTest extends IntegrationBase {
     @Autowired
     private ArmDataManagementConfiguration armDataManagementConfiguration;
     @Autowired
```

Comment:
Can you have @Autowired and @SpyBean on same object?

Reply:
I believe so

Spy bean just wraps the autowired object in a Mockito.spy method so we can run verify commands on them